### DISCUSSION TITLE
Pimcore Academy Statments i18n

### DISCUSSION THREAD

**Discussion by keya86** [pimcore/pimcore#15463] [Q&A / Support]
Hi Guys ,
Im trying to get the junior Developer Certificate on the pimcore Academy but this question breaks me.
Can somebody give me a hint?
"What Statements concerning i18n are correct"
1) One Document can have multiple languages
2) One document always have exactly one languages
3) Multiple languages Websites can be built via multiple documenta trees
4) URLs can be translated with shared Translation
5) Labels in templates can be translated with shared Translation.

I watched the Videos and read the dokumentation, so I checked every Box except the first. But it came back incorrect. Im so confused. Please help.

**Comment by dvesh3** [comment]
@keya86 Hey, I can help you getting closer to the correct answer but first would like to know your understanding about the selected options specially "URLs can be translated with shared Translation". Can you share your thoughts?

**Comment by keya86** [comment]
Hi. 
In the Video that was shown, that he uses clone and the URL was automaticlly translated so i thought this must have happend with shared translation because he said pimcore helps with translating.

Thanks for your help.

**Reply by dvesh3** [reply]
When cloning the document tree, you select the language which is then saved for the new cloned document tree. This language is set in the request locale of current document and the labels are translated automatically on the templates **and that's how Pimcore helps in translating the content not the URL itself** ~ you can exclude this option from your answer.
